logging/src/main/java/org/jboss/as/logging/LoggingExtension.java

```diff
@@ -71,7 +71,9 @@
 
     static final PathElement LOGGING_PROFILE_PATH = PathElement.pathElement(CommonAttributes.LOGGING_PROFILE);
 
-    static final WildFlyLogContextSelector CONTEXT_SELECTOR = WildFlyLogContextSelector.Factory.create();
+    /** @deprecated only present for use by an integration test hack */
+    @Deprecated
+    static WildFlyLogContextSelector CONTEXT_SELECTOR = WildFlyLogContextSelector.Factory.create();
```

Comment:
Just an FYI I filed a JIRA to remove the hacky test that uses this https://issues.jboss.org/browse/WFLY-4410